### PR TITLE
Design fixes: review

### PR DESCRIPTION
**Pull request by kseniaSs**
1. Change padding for input && select
2. Add default value for select
3. Add hover for submit
4. Add resize for textarea
5. Change size arrow

### CODE REVIEW

**Comment by TatianaFomina** [comment]
hi, there are still some small differences from the mock up. Seems like width and paddings should be updated

https://user-images.githubusercontent.com/31101125/219098999-9ce50ba0-56c7-453e-9e01-366696c2bc3a.mov

File: src/styles/form.module.css [hunk 1]
```diff
@@ -24,7 +24,7 @@
 .collapsed {
   display: inline-flex;
   flex-direction: column;
-  padding: 10px;
+  padding: 10px 10px 10px 9px;
```
Comment (neSpecc) [inline src/styles/form.module.css:27]: why `9`? all paddings are `10` in figma

File: src/styles/form.module.css [hunk 2]
```diff
@@ -1,15 +1,15 @@
 @value colors: "./tokens/colors.module.css";
 @value fonts: "./tokens/fonts.module.css";
-@value color-box-shadow, color-placeholder, color-text-white, color-submit, color-select-border, color-border, color-background-primary from colors;
+@value color-box-shadow, color-submit-hover, color-placeholder, color-text-white, color-submit, color-select-border, color-border, color-background-primary from colors;
 @value main-font-size, normal-font-weight, bold-font-weight, main-line-height,  extra-bold-font-weight, main-font-weight from fonts;
 
 .container {
```
Comment (neSpecc) [inline src/styles/form.module.css:6]: lets add font-smoothing to make all typography look better

**Comment by neSpecc** [comment]
The cross icon looks bad on Safari:

image

Also, it has 20x20 size, but it should have 24x24

File: src/styles/form.module.css (hunk 1 above)
Comment (kseniaSs) [reply src/styles/form.module.css:27]: In figma it's includes border. Maybe it is the mistake? 
Снимок экрана 2023-02-26 в 19 18 41

Comment (neSpecc) [reply src/styles/form.module.css:27]: Yea, that's my bad. I've updated Figma to make that border outside.

Also, I've updated the border color to make it the same as the inputs have.

**Comment by neSpecc** [comment]
The clickable zone of a cross is too small (20x20) and has the wrong alignment:
image

It should be 24x24 with the cross at the center, see Figma mockup:

image

**Review by neSpecc — APPROVED** [review]

**Review by TatianaFomina — APPROVED** [review]

**Pull request merged**